google/cloud/bigquery/_versions_helpers.py

```diff
@@ -234,3 +237,10 @@ def try_import(self, raise_if_error: bool = False) -> Any:
 
 
 PANDAS_VERSIONS = PandasVersions()
+
+SUPPORTS_RANGE_PYARROW = (
```

Comment:
I wonder if we should make this a property of `PANDAS_VERSIONS` or `PYARROW_VERSIONS` 🤔 

I suppose since it requires both, it makes sense to pull it separate. Let's add a comment explaining that.

```suggestion
# Since RANGE support in pandas requires specific versions
# of both pyarrow and pandas, we make this a separate
# constant instead of as a property of PANDAS_VERSIONS
# or PYARROW_VERSIONS.
SUPPORTS_RANGE_PYARROW = (
```

Reply:
Thanks for the suggestion!